pallets/identity/src/benchmarking.rs

```diff
@@ -20,6 +20,7 @@ pub fn get_origin<T: Config>(name: &'static str) -> RawOrigin<T::AccountId> {
     RawOrigin::Signed(get_account::<T>(name))
 }
 
+// SBP-M1 review: I am not sure if 100 elements are enough for proper benchmarking...
 benchmarks! {
     create_identity {
         let s in 0 .. 100;
```

Comment:
```suggestion
        let s in 0 .. 100000;
```
Is there a threshold that would be preferable? My gut says to increase by a couple OOMs.